tp-w8961n-status.py

```
@@ -88,14 +90,14 @@ def parse(self, response):
+            print(time.strftime('%Y-%m-%d %H:%M-%S'))
+            with TPW8961N(Config.ROUTER_IP) as modem:
+                print(modem.send_cmd(StatusCommand()))
+                print(modem.send_cmd(RateCommand()))
+                print(modem.send_cmd(QualityCommand('downstream')))
+                print(modem.send_cmd(QualityCommand('upstream')))
+            time.sleep(Config.POLLING_INTERVAL_SECONDS)
         except KeyboardInterrupt:
             raise
-        except e:
-            print e
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)